src/main/java/org/gridsuite/directory/server/dto/ElementAttributes.java

```diff
@@ -52,7 +52,7 @@ public class ElementAttributes {
     private String lastModifiedBy;
 
     public boolean isAllowed(@NonNull String userId) {
-        if (!type.equals(DIRECTORY)) {
+        if (!type.equals(ElementType.DIRECTORY.name())) {
```

Comment:
Why not using the enum  type `ElementType` for `type` member instead of `String` then you avoid calling .name() everywhere no ?